synapse/lib/cell.py

```diff
@@ -2954,9 +3103,9 @@ async def _initCellHiveAuth(self):
 
         seed = s_common.guid((self.iden, 'hive', 'auth'))
 
-        node = await self.hive.open(('auth',))
-        auth = await s_hiveauth.Auth.anit(
-            node,
+        auth = await s_auth.Auth.anit(
```

Comment:
We're allowed to change `_initCellHiveAuth` to `_initCellAuth` since we're not longer using the hive.